### PR TITLE
Fixed grammar, upped consistency, fixed typo/bad link in Labels section.

### PR DESCRIPTION
**Pull request by BurkHufnagel**
Fixed grammar, upped style consistency, fixed a typo/bad link in the Labels section.
Broke long (maybe run-on) sentences into shorter ones to make them easier to read and understand.
Note: There is a new comment describing a recommended change to the image containing a description of the three phases for making contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,18 @@ with **simple steps** people can **_easily_ follow** <br />
...
 
 Our contribution steps in a _nutshell_:
-
+<!-- The text under the magnifying glass says: 
+        "validate the need exists"
+        "or undesired behavior"
+        "from the user's perspective"
+
+      I believe this should be something like
+        "validate the need or"
```
Comment (nelsonic) [inline README.md:19]: Agreed. the text under the magnifying glass is not _complete_.
validate-idea
The image was created in Paint. 😜 
If you have _any_ image editor, grab the original and modify it. 
(_not urgent for **`this`** PR, but if you're feeling that way inclined in future..._) 👍

File: README.md [hunk 2]
```diff
@@ -60,7 +71,7 @@ and **worked on** as a **team**_ while keeping ***everyone informed***. <br />
 We use _issues_ to _log_ our user stories, sub/technical tasks
 and any ***progress*** towards solving the issue, <br />
 this ***ensures*** we have a
-[_**single source** of the **truth**_](https://github.com/dwyl/github-reference#why)
+[_**single source** of **truth**_](https://github.com/dwyl/github-reference#why)
```
Comment (nelsonic) [inline README.md:74]: 👍

File: README.md [hunk 3]
```diff
@@ -69,27 +80,27 @@ that _everyone_ can easily see!
...
+1. If you ***find*** an _existing_ issue that matches what you were looking for,
 _please_ click on the title of the issue, ***read the description*** to
-confirm if it's what you are looking for, then ***comment*** on it
+confirm that it's what you are looking for, then ***comment*** on it
 so we know more than one person has the need/issue. <br />
-> _**Note**: if there is **already an issue** for what you were thinking,
+> _**Note**: If there is **already an issue** for what you were thinking,
 you can **skip** steps 2 & 3.
-as there's **no need** to create a **new issue**_.
+as there's **no need** to create a **duplicate new issue**_.
```
Comment (nelsonic) [inline README.md:100]: ➕  addition of **duplicate** is good. thanks. 👍

File: README.md [hunk 4]
```diff
@@ -343,19 +353,19 @@ this gives _everyone_ in the team a **_clear_ picture** of what is ***planned***
   ["***Readme Driven Development***"](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)
   i.e: you ***document*** your work
   ***before*** you write ***any code***. <br />
-  (_if this feels "tedious" to you, ask yourself
+  (_If this feels "tedious" to you,
```
Comment (nelsonic) [inline README.md:356]: thanks for tidying up this incoherent sentence. 👍

**Review by nelsonic — APPROVED** [review]
@BurkHufnagel thank you so much for taking the time to read the Epically long Readme so well that you were able to make all these improvements. 🎉

**Comment by BurkHufnagel** [comment]
You're welcome, and thanks for the compliments.]

"Any sufficiently analyzed magic is indistinguishable from science!" Agatha
Heterodyne (Girl Genius)

**Pull request merged**